packages/vsce/__tests__/__e2e__/01_EditProjectTeamConfiguration.test.ts

```diff
@@ -88,4 +89,5 @@ describe("Create Project Level Team Configuration File Scenario", () => {
 
     cicsTree.takeScreenshot();
   });
+  resetAllScenarios()
```

Comment:
Thinking we should put this in an `after` block, just to be consistent with the test infrastructure setup etc.

```suggestion
  after(() => {
  resetAllScenarios()
  })
```